src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -342,35 +343,37 @@ export const MoreActionsDrawer = ({
               </ToggleButton>
             </ListItem>
 
-            <ListItem disablePadding={false}>
+            <ListItem disablePadding>
               <Typography variant="body2">Layout Thoroughness</Typography>
               <Tooltip
-                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. 1 = lowest effort, 100 = highest effort."
-                enterTouchDelay={0} // allow touch to immediately trigger
-                leaveTouchDelay={Infinity} // touch-away to close on mobile, since message is long
+                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. Low = minimal effort, High = maximum effort."
+                enterTouchDelay={0} // Trigger immediately on touch
+                leaveTouchDelay={Infinity} // Close on touch-away
               >
                 <IconButton
                   color="info"
                   aria-label="Thoroughness info"
                   sx={{
-                    // Don't make it look like clicking will do something, since it won't.
-                    // Using a button here is an attempt to make it accessible, since the tooltip will show
-                    // on focus.
-                    cursor: "default",
+                    cursor: "default", // Maintain accessibility while preventing click actions
                     alignSelf: "center",
                   }}
                 >
                   <Info />
                 </IconButton>
               </Tooltip>
-              <NumberInput
-                min={1}
-                max={100}
+              <Select
                 value={layoutThoroughness}
-                onChange={(_event, value) => {
-                  if (value) setLayoutThoroughness(value);
+                onChange={(event) => setLayoutThoroughness(Number(event.target.value))}
+                fullWidth // Use fullWidth for proper alignment
+                size="small" // Smaller size for better fit
+                sx={{
+                  marginLeft: "16px", // Adjust spacing
```

Comment:
The dropdown is visually separated from the info icon already, so this `marginLeft` isn't necessary (Mui buttons give themselves padding by default)